--- seafile-5.0.0-server.orig/scripts/seaf-gc.sh	2015-11-02 06:46:49.392074390 +0800
+++ seafile-5.0.0-server.orig/scripts/seaf-gc.sh	2015-11-02 06:51:42.890466374 +0800
@@ -7,11 +7,10 @@
 TOPDIR=$(dirname "${INSTALLPATH}")
 default_ccnet_conf_dir=${TOPDIR}/ccnet
 default_conf_dir=${TOPDIR}/conf
-seaf_gc=${INSTALLPATH}/seafile/bin/seafserv-gc
+seaf_gc=/usr/bin/seafserv-gc
 seaf_gc_opts=""
 
-export PATH=${INSTALLPATH}/seafile/bin:$PATH
-export SEAFILE_LD_LIBRARY_PATH=${INSTALLPATH}/seafile/lib/:${INSTALLPATH}/seafile/lib64:${LD_LIBRARY_PATH}
+export PYTHONPATH="/usr/share/seafile/seafile-server/seahub/thirdpart:$PYTHONPATH"
 
 script_name=$0
 function usage () {
@@ -79,11 +78,7 @@
 
     echo "Starting seafserv-gc, please wait ..."
 
-    LD_LIBRARY_PATH=$SEAFILE_LD_LIBRARY_PATH ${seaf_gc} \
-        -c "${default_ccnet_conf_dir}" \
-        -d "${seafile_data_dir}" \
-        -F "${default_conf_dir}" \
-        ${seaf_gc_opts}
+    ${seaf_gc} -c "${default_ccnet_conf_dir}" -d "${seafile_data_dir}" ${seaf_gc_opts}
 
     echo "seafserv-gc run done"
     echo
--- seafile-5.0.0-server.orig/scripts/setup-seafile-mysql.sh	2015-11-02 06:56:44.885869706 +0800
+++ seafile-5.0.0-server.orig/scripts/setup-seafile-mysql.sh	2015-11-02 06:57:11.719905545 +0800
@@ -10,6 +10,7 @@
 INSTALLPATH=$(dirname "${SCRIPT}")
 
 cd "$INSTALLPATH"
+export PYTHONPATH="/usr/share/seafile/seafile-server/seahub/thirdpart:$PYTHONPATH"
 
 python_script=setup-seafile-mysql.py
